content/en/docs/faq.md

```diff
@@ -75,3 +75,34 @@ Not natively supported but a user can configure `revocationValidations` to `skip
+
+**Q: How are Notary and Notary Project related?**
+
+**A:** Notary, a tool for artifact signing based on The Update Framework (TUF), is developed by the Notary Project community and is hosted within the Notary Project's GitHub organization. So, notary is a tool, and Notary Project is the community that developed and maintains it.
+
+**Q: What is the difference between Notary Project and TUF?**
+
+**A:** The Notary Project, as a [GitHub organization](https://github.com/notaryproject), host multiple repositories including a [TUF](https://github.com/notaryproject/tuf) repository, which is a specific repository intended for prototyping the storage of TUF metadata in OCI-compliant registries.
+
+**Q: What is Notary Project specification?**
+
+**A:** Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary Project tooling.
+
+**Q: Does Notary Project signature specification leverage TUF?**
+
+**A:** 
+The Notary Project [signature specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md) does not leverage TUF. Instead, it leverages [COSE](https://github.com/notaryproject/specifications/blob/main/specs/signature-envelope-cose.md) and [JWS](https://github.com/notaryproject/specifications/blob/main/specs/) envelopes to produce and handle portable signatures. 
+
+**Q: What is the difference between Notary and Notation?**
+
+**A:** Notary and Notation are both solutions for signing artifacts. The main difference between these two tools is that notary is based on [The Update Framework](https://theupdateframework.com) (TUF) specification while Notation is non-TUF-based. While notary has server and client components, Notation has a CLI and libraries. An example of a notary implementation is Docker Content Trust (DCT). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
+
+**Q: I've heard the term "Notary v2". What does this mean?**
+
+**A:** The term "notary v2" was previously used by the Notary Project and others. However, various meanings were ascribed to it, leading to its ambiguous usage with some people referring to it as the entire Notary Project and others as the Notation CLI. Because of this ambiguity, the term "notary v2" is no longer used by the Notary Project. While the term may still be visible in some articles on the internet, the name "notary v2" is no longer valid.
```

Comment:
```suggestion
**A:** The term "Notary v2" or "notary v2" was previously used by the Notary Project and others. However, various meanings were ascribed to it, leading to its ambiguous usage with some people referring to it as the entire Notary Project and others as the [Notation CLI](https://github.com/notaryproject/notation). Because of this ambiguity, the term "Notary v2" or "notary v2" is no longer used by the Notary Project. While the term may still be visible in some articles on the internet, the name "Notary v2" or "notary v2" is no longer valid.
```